app/domain/logs.rb

```diff
@@ -188,6 +198,40 @@ module AuthnAzure
       )
 
     end
+
+    module AuthnGce
+
+      ExtractingRestrictionsFromResource = ::Util::TrackableLogMessageClass.new(
+        msg: "Extracting resource restrictions with prefix '{0-prefix}' from role '{1-role-id}'",
+        code: "CONJ00039D"
+      )
+
+      ExtractedResourceRestrictions = ::Util::TrackableLogMessageClass.new(
+        msg: "{0-amount} resource restrictions were extracted",
+        code: "CONJ00040D"
+      )
+
+      ValidatingResourceRestrictionsConfiguration = ::Util::TrackableLogMessageClass.new(
+        msg: "Validating resource restrictions configuration",
+        code: "CONJ00041D"
+      )
+
+      ValidatedResourceRestrictionsConfiguration = ::Util::TrackableLogMessageClass.new(
+        msg: "Resource restrictions configuration is validated",
+        code: "CONJ00042D"
+      )
+
+      ValidatingResourceRestrictionsValues = ::Util::TrackableLogMessageClass.new(
+        msg: "Validating resource restrictions values against Google ID token",
+        code: "CONJ00044D"
+      )
+
+      ValidatedResourceRestrictionsValues = ::Util::TrackableLogMessageClass.new(
+        msg: "Resource restrictions values are validated",
+        code: "CONJ00045D"
```

Comment:
"Resource restriction values validated"

no "s" on "restriction", and removed "are"